src/es/manhuaonline/src/eu/kanade/tachiyomi/extension/es/manhuaonline/SamuraiScan.kt

```diff
@@ -17,7 +17,7 @@ class SamuraiScan : Madara(
     override val useLoadMoreRequest = LoadMoreStrategy.Never
     override val useNewChapterEndpoint = true
 
-    override val mangaSubString = "read"
+    override val mangaSubString = "leer"
```

Comment:
Would using `rd` be more correct? It would avoid a redirect in `fetchSearchManga()`

Reply:
Yes actually, already changed